docs/book/src/component-config-tutorial/testdata/project/api/v2/zz_generated.deepcopy.go

```diff
@@ -1,51 +0,0 @@
-// +build !ignore_autogenerated
```

Comment:
After update the projects in the docs, could you please call the target make all to ensure that we will update/generate all manifest?